src/components/CategoryList/CategoryList.tsx

```diff
@@ -19,9 +21,11 @@ export const CategoryList: React.FC<CategoryListProps> = ({
   primary,
   categories,
   selectCategory,
+  pendingStatus
 }) => {
   return (
     <CategoryListContainer>
+      {pendingStatus === 'loading' && <PendingIndicator alt={"Pending indicator"} size={64} />}
```

Comment:
as size use enum IconSize

Reply:
We are doing lot of checks against the current status and its also used in slices.
Please create folder with file constants/index.ts
export enum FETCH_STATUS {
LOADING: 'loading'
IDLE:'idle' 
}

Replace all hardcoded strings and use enum (slices, components)